tpm/ak.go

```diff
@@ -136,7 +135,7 @@ func (t *TPM) CreateAK(ctx context.Context, name string) (*AK, error) {
 	}
 
 	if _, err := t.store.GetAK(name); err == nil {
-		return nil, fmt.Errorf("failed creating AK %q: %w", name, ErrExists)
+		return nil, fmt.Errorf("failed getting AK %q: %w", name, ErrExists)
```

Comment:
The error is creating the key, not getting it. It cannot create the key because it exists. Getting the key does not fail.